core/templates/pages/exploration-editor-page/editor-tab/templates/modal-templates/customize-interaction-modal.component.ts

```diff
@@ -254,10 +256,23 @@ export class CustomizeInteractionModalComponent
   }
 
   isSaveInteractionButtonEnabled(): boolean {
+    let exploration = this.getTitle(this.stateInteractionIdService.displayed);
+    if (exploration === 'End Exploration') {
+      let inputField = document.getElementsByClassName('form-control display-inline');
```

Comment:
These class names are too vague and other elements could have the same classes which could cause conflicts. Please add a specific class name to this element and access it via that.

Reply:
added a new class. I have also checked that that class is not used somewhere else anywhere